[2023-03-29T18:56:56.614+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.store_user manual__2023-03-29T18:56:50.384187+00:00 [queued]>
[2023-03-29T18:56:56.622+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.store_user manual__2023-03-29T18:56:50.384187+00:00 [queued]>
[2023-03-29T18:56:56.623+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-03-29T18:56:56.624+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-03-29T18:56:56.626+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-03-29T18:56:56.637+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): store_user> on 2023-03-29 18:56:50.384187+00:00
[2023-03-29T18:56:56.643+0000] {standard_task_runner.py:55} INFO - Started process 17953 to run task
[2023-03-29T18:56:56.647+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'store_user', 'manual__2023-03-29T18:56:50.384187+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpz6z8cdbx']
[2023-03-29T18:56:56.650+0000] {standard_task_runner.py:83} INFO - Job 25: Subtask store_user
[2023-03-29T18:56:56.711+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.store_user manual__2023-03-29T18:56:50.384187+00:00 [running]> on host 9dcb783b1f3a
[2023-03-29T18:56:56.775+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=store_user
AIRFLOW_CTX_EXECUTION_DATE=2023-03-29T18:56:50.384187+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-29T18:56:50.384187+00:00
[2023-03-29T18:56:56.778+0000] {postgres.py:129} INFO - Running copy expert: COPY users FROM stdin WITH DELIMITER as ',', filename: /tmp/processed_user.csv
[2023-03-29T18:56:56.791+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-03-29T18:56:56.798+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-03-29T18:56:56.813+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=store_user, execution_date=20230329T185650, start_date=20230329T185656, end_date=20230329T185656
[2023-03-29T18:56:56.861+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-03-29T18:56:56.881+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
